migrations/versions/205742d3b3f5_change_blacklist_to_deny.py

```diff
@@ -0,0 +1,31 @@
+"""Change blacklist to deny
+
+Revision ID: 205742d3b3f5
+Revises: 4b6587b98b26
+Create Date: 2020-07-04 19:29:40.607069
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '205742d3b3f5'
+down_revision = '4b6587b98b26'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('key', sa.Column('denied', sa.Boolean(), nullable=True))
+    op.drop_column('key', 'blacklisted')
```

Comment:
you should probably add a migration to populate the new column with the old values. same with the downgrade